src/targetSelector.ts

```diff
@@ -141,6 +143,27 @@ export class TargetSelector {
     return result;
   }
 
+  public async showBuildUseCase() {
```

Comment:
Maybe this should rather be part of the app selector ? Not much to do with targets

Reply:
Done